fix(Select / Combobox / Dropdown / DatePicker): Menu closing support with escape key in Modals. (#1082)
cengage/react-magma: .changeset/fix-modalEscape.md

---
'react-magma-dom': patch
---

> fix(Modal): Added support for closing components that have menus with the escape key inside of modals while retaining an open modal state.

Reviewer: Can we list all the components this affects? This is actually not a bug on the modal but on the individual components. Our adopters will want to know what components were changed in each version in case they run into problems.

> fix(Modal): Added support for closing components that have menus with the escape key inside of modals while retaining an open modal state. The components affected are Select, Combobox, and Dropdown.

Reviewer: Typically the components affected by the change are what we put in parenthesis:
```suggestion
fix(Select/Combobox/Dropdown/DatePicker): Added support for closing components that have menus with the escape key inside of modals while retaining an open modal state.
```

Reviewer: And we should update the title of the PR to match this description
